src/Controller/SentencesListsController.php

```diff
@@ -365,6 +367,27 @@ public function add_new_sentence_to_list()
         $this->set('sentence', $result);
     }
 
+    public function add_sentence_to_new_list() {
+        $userId = $this->Auth->user('id');
+        $sentenceId = $this->request->getData('sentenceId');
+        $list = $this->SentencesLists->createList(
+            $this->request->getData('name'), 
+            $this->Auth->user('id')
+        );
```

Comment:
`createList()` will return false if the list name is invalid (empty or empty after trimming). However, you are not checking for this case.